hedera-node/src/test/java/com/hedera/services/txns/contract/ContractCallTransitionLogicTest.java

```diff
@@ -74,15 +92,23 @@
 	private HederaWorldState worldState;
 	private UsagePricesProvider usagePrices;
 	private TransactionRecordService recordService;
+	private HederaWorldState.Updater updater;
 
 	MerkleMap<EntityNum, MerkleAccount> contracts;
 	ContractCallTransitionLogic subject;
 
+	private ExchangeRate exchangeRate;
+	private Transaction transaction;
```

Comment:
I think we don't use this Transaction type in EvmTxProcessor anymore, do we still need to mock it?